examples/job/generate-slabs-al2o3.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Overview
+# 
+# This example has been created as part of our Advanced Topics Webinar, scheduled for March 26, 2021. This webinar focused on the high-throughput screening of surfaces using Exabyte.
```

Comment:
Maybe "generate-slab-structures-and-relax-them"?

Reply:
We do not relax the slabs generated in this notebook because some of them were extremely large. Renamed to `generate-al2o3-slab-structures`